apps/cowswap-frontend/src/common/hooks/useOrdersPanel.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useCallback } from 'react';
```

Comment:
creating a reusable hook to control the orders panel (user activity modal)

Reply:
Refactored it into `accountModalStateAtom`